configure.ac

```diff
@@ -1383,6 +1393,35 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
     AC_MSG_RESULT(no)
 ])
 
+have_libxdp=no
+dnl Check for LIBXDP AF_XDP socket support
+AC_MSG_CHECKING(for LIBXDP XDP packet sending support)
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
+#include <stdlib.h>
+#include <xdp/xsk.h>
+]], [[
+    struct xsk_socket {
+        struct xsk_ring_cons *rx;
+        struct xsk_ring_prod *tx;
+        struct xsk_ctx *ctx;
+        struct xsk_socket_config config;
+        int fd;
+    };
+    struct xsk_socket *xsk;
+	struct xsk_ring_cons *rxr = NULL;
+	struct xsk_ring_prod *txr = NULL;
+	xsk = (struct xsk_socket*)malloc(sizeof(struct xsk_socket));
+    int queue_id = 0;
+    xsk_socket__create(xsk, "lo", queue_id, NULL, rxr, txr, NULL);
```

Comment:
Same as before. xsk_socket__create's might fail despite the fact that xsk_socket__create is exits and compile.

Reply:
unfortunately this code will not be executed, we are checking only the compilation process